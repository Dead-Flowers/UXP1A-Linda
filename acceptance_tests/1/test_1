-t 1 output "(123)"
-t 1 input "(integer:*)"
