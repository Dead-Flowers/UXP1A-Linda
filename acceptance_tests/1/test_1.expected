Exit code: 0
(123)
Exit code: 0
